bash_completion

```diff
@@ -2567,7 +2567,7 @@ __load_completion()
             # Do not warn with . or .. (especially the former is common)
             [[ $compfile == */.?(.) ]] ||
                 echo "bash_completion: $compfile: is a directory" >&2
-        elif [[ -e $compfile ]] && . "$compfile"; then
+        elif [[ -e $compfile ]] && . "$compfile" "$cmd"; then
```

Comment:
I guess this is something we could consider adding a test for, but I'm not sure if it's worth the trouble.

Also not sure about adding tests for the the actual completions as their result varies depending on if the generating command is installed or not.